src/components/measureLanding/measureList/MeasureList.tsx

```diff
@@ -301,6 +303,119 @@ export default function MeasureList(props: {
       enableSorting: false,
     },
   ];
+  const columnsBehindFlag = [
+    {
+      header: "Measure",
+      cell: (info) => (
+        <>
+          <TruncateText
+            text={info.row.original.measureName}
+            maxLength={120}
+            dataTestId={`measure-name-${info.row.original.id}`}
+          />
+        </>
+      ),
+      accessorKey: "measureName",
+      sortingFn: (rowA, rowB) =>
+        customSort(rowA.original.measureName, rowB.original.measureName),
+    },
+    {
+      header: "Version",
+      cell: (info) => (
+        <TruncateText
+          text={info.row.original.version}
+          maxLength={60}
+          dataTestId={`measure-version-${info.row.original.id}`}
+        />
+      ),
+      accessorKey: "version",
+      sortingFn: (rowA, rowB) =>
+        customSort(rowA.original.version, rowB.original.version),
+    },
+    {
+      header: "Status",
+      cell: (info) => (
+        <>
+          {`${info.row.original.actions.measureMetaData?.draft}` === "true" && (
+            <Chip tw="ml-6" className="chip-draft" label="Draft" />
```

Comment:
I think you want to remove the `tw="ml-6"`. Adding the margin is making that column a bit crooked.